lib/fiware-iotagent-lib.js

```diff
@@ -305,3 +361,4 @@ exports.constants = constants;
 
 exports.logModule = logger;
 exports.configModule = config;
+exports.startServer = startServer;
```

Comment:
I think the new function should be included in this section of the documentation https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/usermanual.md#function-reference, using the same style as other functions exposed by this module.

Reply:
Fixed in 4a5eeb2